saleor/checkout/base_calculations.py

```diff
@@ -38,148 +78,82 @@ def _calculate_base_line_unit_price(
-            line_info.product,
-            line_info.collections,
-            channel,
-            line_info.channel_listing,
-            [],
-            line_info.line.price_override,
-        )
-
     if line_info.voucher and not line_info.voucher.apply_once_per_order:
-        price_with_discounts = max(
+        unit_price = max(
             variant_price
             - line_info.voucher.get_discount_amount_for(variant_price, channel=channel),
             zero_money(variant_price.currency),
         )
     else:
-        price_with_discounts = variant_price
-
-    return CheckoutPricesData(
-        undiscounted_price=quantize_price(
-            undiscounted_variant_price, undiscounted_variant_price.currency
-        ),
-        price_with_sale=quantize_price(variant_price, variant_price.currency),
-        price_with_discounts=quantize_price(
-            price_with_discounts, price_with_discounts.currency
-        ),
-    )
+        unit_price = variant_price
 
+    return quantize_price(unit_price, unit_price.currency)
 
-def calculate_base_line_unit_price(
-    line_info: CheckoutLineInfo,
+
+def calculate_undiscounted_base_line_total_price(
+    line_info: "CheckoutLineInfo",
     channel: "Channel",
-    discounts: Optional[Iterable[DiscountInfo]] = None,
-) -> CheckoutPricesData:
-    """Calculate line unit prices including discounts and vouchers.
-
-    Returns a three money values. Undiscounted price, price and price with voucher.
-    Voucher is added to 'price_with_discounts' when line's product matches to products
-    applicable for voucher.
-    For voucher with 'apply once per order', the price will be included in unit price.
-    'price' includes discount from sale if any valid exists.
-    'price_with_discounts' includes voucher discount and sale discount if any valid
-    exists.
-    'undiscounted_price' is a price without any sale and voucher.
-    """
-    prices_data = calculate_base_line_total_price(
-        line_info=line_info, channel=channel, discounts=discounts
-    )
-    quantity = line_info.line.quantity
-    currency = prices_data.price_with_discounts.currency
-    return CheckoutPricesData(
-        undiscounted_price=quantize_price(
-            prices_data.undiscounted_price / quantity, currency
-        ),
-        price_with_sale=quantize_price(
-            prices_data.price_with_sale / quantity, currency
-        ),
-        price_with_discounts=quantize_price(
-            prices_data.price_with_discounts / quantity, currency
-        ),
+) -> Money:
+    """Calculate line total price excluding discounts and vouchers."""
+    unit_price = _calculate_undiscounted_base_line_unit_price(
+        line_info=line_info, channel=channel
     )
+    total_price = unit_price * line_info.line.quantity
+    return quantize_price(total_price, total_price.currency)
 
 
-def calculate_base_line_total_price(
-    line_info: CheckoutLineInfo,
+def calculate_undiscounted_base_line_unit_price(
```

Comment:
do we need this function now since all it does is calling `_calculate_undiscounted_base_line_unit_price` with the same parameters?